### ISSUE TITLE
AIT2 Experience and Cost

### ISSUE THREAD

**Issue by justlstn** [aptos-labs/aptos-ait2#68]
### Summary
Basically installation and maintaining process was gone pretty easy for me. It was easy for me to follow step-by-step installation & maintaining guide from the documentation. 

During AIT2 there were some errors in the log that were confusing everyone, for example `Peer not connected` or [this one](https://github.com/aptos-labs/aptos-ait2/issues/15) but got answers that it's okay. In general I didn't experience some sync/connection issues probably because my server is at good location and the network bandwidth were enough to sync.

The only case I'd like to mention here was that after updating `validator.yaml` file with `round_initial_timeout_ms: 2000` vote / hour rate was decreased significant but proposals counter increased in a moment so that after I started up a container the proposals / hour rate was bigger than vote / hour. But later the proposals / hour rate also dropped so that I moved from 22 place on 24 place on the dashboard.

In general, during AIT2 my Node liveness & Participation were at 100%, although according to Grafana dashboard the Uptime was 99.5% and about 3-4 days before the end of AIT2 it was up to 99.6%.

### Server & validator node characteristics

**CPU**: Coffee Lake-S, Intel(R) Xeon(R) E-2276G, 6 cores / 12 threads @ 3.8 GHz (Turbo 4.9 GHz)
**RAM**: 64 GB ECC DDR4 RAM
**Drives**: 2 x 960 GB NVMe SSD Datacenter Edition
**Location**: Finland, Helsinki
**Network bandwidth**: 1Gbit
**Price per month**: € 74
**OS**: Debian
**Env**: Docker

In conclusion, I'd like to say the AIT2 is really good experience for me, as I mentioned before there weren't some major difficulties during AIT2, the community is really great! Thanks for the opportunity to participate in the event!
